Kitodo-DataAccess/src/main/java/org/kitodo/dataaccess/format/xml/Space.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess.format.xml;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Enum implementing the XML white space handling value constants.
+ *
+ * @see "http://www.w3.org/TR/xml/#sec-white-space"
+ */
+enum Space {
+
+    /**
+     * Signals that white-space processing is acceptable for this element.
+     */
+    DEFAULT("default") {
+        @Override
+        /**
+         * Trimming white space the XML way.
+         */
+        public String trim(String s) {
```

Comment:
Short method parameter names should be avoided. Please use a longer and understandable name.